### PR TITLE
QBI query component filtering

### PR DESCRIPTION
**Pull request by paulbalaji**
#### Description

Add component filters to qbi queries

drive-by upgrade to latest entities package too

#### Tests

- [x] local dpl
- [x] cloud dpl

#### Documentation

- [x] changelog

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.

### CODE REVIEW

File: workers/unity/Assets/Fps/Scenes/FPS-Development.unity [hunk 1]
```diff
@@ -559,6 +559,11 @@ PrefabInstance:
       propertyPath: m_Enabled
       value: 1
       objectReference: {fileID: 0}
+    - target: {fileID: 114510834623507516, guid: a18c738cbcda9914591d9af24a618907,
```
Comment (zeroZshadow) [inline workers/unity/Assets/Fps/Scenes/FPS-Development.unity:562]: What is this?

File: workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs [hunk 2]
```diff
@@ -90,16 +82,38 @@ public static EntityTemplate Player(EntityId entityId, string workerId, byte[] a
...
+            var clientSelfInterest = InterestQuery.Query(Constraint.EntityId(entityId)).FilterResults(new[]
+            {
+                Position.ComponentId, Metadata.ComponentId, OwningWorker.ComponentId,
+                ServerMovement.ComponentId, HealthComponent.ComponentId, GunComponent.ComponentId
+            });
+
+            var clientRangeInterest = InterestQuery.Query(Constraint.RelativeCylinder(clientRadius)).FilterResults(new[]
+            {
+                Position.ComponentId, Metadata.ComponentId, OwningWorker.ComponentId,
+                ServerMovement.ComponentId, ClientRotation.ComponentId, HealthComponent.ComponentId,
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs:94]: Do players need other player's health?

File: workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs [hunk 3]
```diff
@@ -90,16 +82,38 @@ public static EntityTemplate Player(EntityId entityId, string workerId, byte[] a
...
+            {
+                ClientMovement.ComponentId, ShootingComponent.ComponentId
+            });
+
+            var serverRangeInterest = InterestQuery.Query(Constraint.RelativeCylinder(serverRadius)).FilterResults(new[]
+            {
+                Position.ComponentId, Metadata.ComponentId, OwningWorker.ComponentId,
+                ServerMovement.ComponentId, ClientRotation.ComponentId, HealthComponent.ComponentId,
+                ShootingComponent.ComponentId
+            });
```
Comment (jamiebrynes7) [inline workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs:108]: I'm having a little trouble understanding why we chose these sets for the server interest, a comment could help? :)

File: workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs (hunk 2 above)
Comment (paulbalaji) [reply workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs:94]: yup, `RagdollSpawner` requires it

File: workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs (hunk 3 above)
Comment (paulbalaji) [reply workers/unity/Assets/Fps/Scripts/Config/FpsEntityTemplates.cs:108]: sure, will add

**Pull request merged**